text-ui-test/input.txt

```diff
@@ -0,0 +1,10 @@
+todo read book
+deadline return book /by June 6th
+event project meeting /from Aug 6th 2pm /to 4pm
+todo join sports club
+mark 1
+mark 4
+todo borrow book
+list
+deadline return book /by Sunday
+event project meeting /from Mon 2pm /to 4pm
```

Comment:
I like that you cover most of the commands, could also add unmark to be even better!